test/gemm-microkernel-tester.h

```diff
@@ -511,28 +545,28 @@ struct GemmTestParams {
         isa_check(isa_check) {}
 
   // Setters for the loops over `k`, `m`, and `n`.
-  GemmTestParams& loop_k(size_t from, size_t to, size_t step = 1) {
-    loop_k_ = LoopParams(from, to, step);
+  GemmTestParams& loop_k(size_t from, size_t to, size_t step = 1, LoopStepType step_type = LoopStepType::NextPrime) {
+    loop_k_ = LoopParams(from, to, step, step_type);
     return *this;
   }
-  GemmTestParams& loop_m(size_t from, size_t to, size_t step = 1) {
-    loop_m_ = LoopParams(from, to, step);
+  GemmTestParams& loop_m(size_t from, size_t to, size_t step = 1, LoopStepType step_type = LoopStepType::Linear) {
+    loop_m_ = LoopParams(from, to, step, step_type);
     return *this;
   }
-  GemmTestParams& loop_n(size_t from, size_t to, size_t step = 1) {
-    loop_n_ = LoopParams(from, to, step);
+  GemmTestParams& loop_n(size_t from, size_t to, size_t step = 1, LoopStepType step_type = LoopStepType::NextPrime) {
```

Comment:
I was thinking we always default to linear and at the callsite we change enum to improve visibility. It should be same semantically just that the test code will be more explicit / readable about these jumps. IMO that's like the added benefit of your introduction of this enum. WDYT?